### PR TITLE
[2.19.x] DDF-UI-273 Theme Zoom Percentage Formatting Issues

### PR DESCRIPTION
**Pull request by abel-connexta**
#### What does this PR do?
Issues:
1. Search Forms - Title input form partially hidden
2. Result Forms - Attributes Title goes underneath its form
3. For both Result and Search Form - When have multiple items and zoomed percentage is higher than 150%, no way to get to other items, since can't scroll down

#### Who is reviewing it? 

@andrewzimmer 
@hayleynorton
@cassandrabailey293 
@zta6 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@mojogitoverhere
#### How should this be tested?

	Issue 1:
	1. Go to Search Forms
	2. Click on "+ Search Forms" to add new search form
	3. Change Theme Zoom Percentage and Spacing to various settings
	4. Ensure the " Search Name" input field is fully visible, no matter the theme settings.
	
	Issue 2:
	1. Go to Result Forms
	2. Change Theme Zoom Percentage and Spacing to various settings 
	3. Click on "New Result Forms" to add new result form
	4. Ensure the "Attributes" input field label is fully visible, no matter the theme settings.
	5. Select "Toggle All" for Attributes drop down, then click save
	6. Open previously saved Result form and alter Theme. Ensure "Attributes" input field label is fully visible, no matter the theme settings.
	
	Issue 3:
	1. Go to Search Forms
	2. Create 6 or more forms
	3. Change Theme Zoom Percentage to max (200%) and Spacing to various settings
	4. Ensure you are able to scroll down to see all the saved form items
	5. Do steps 1 to 4 for Result Forms
#### What are the relevant tickets?
codice/ddf-ui#273

##### Before
Search Forms
Result Forms
After zoom - Search Forms
After zoom - Result Forms

##### After
After - Search Form Editor
After - Result Form Editor

Search-Form-Scroll
Result-Forms-Scroll
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

**Comment by abel-connexta** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/component/tabs/tabs.less [hunk 1]
```diff
@@ -118,12 +118,12 @@
...
     height: ~'calc(100% - @{minimumButtonSize})';
     background: inherit;
+    overflow: auto;
 
     > * {
       margin-top: 1px;
     }
     > div {
-      height: 100%;
     }
```
Comment (mojogitoverhere) [inline ui/packages/catalog-ui-search/src/main/webapp/component/tabs/tabs.less:127]: Should we remove this selector now that it is empty?

**Comment by andrewzimmer** [comment]
build now

**Comment by andrewzimmer** [comment]
Hero Successful 🎉 
- Search and Result forms can be fully accessed at all zoom levels and theme combinations

**Comment by andrewzimmer** [comment]
build now

**Pull request merged**